flow/platforms/ihp-sg13g2/pdn.tcl

```diff
@@ -21,4 +25,7 @@ add_pdn_stripe -grid {grid} -layer {Metal5}     -width {2.200} -pitch {75.6} -of
 add_pdn_stripe -grid {grid} -layer {TopMetal1}  -width {1.800} -pitch {75.6} -offset {13.570}             -extend_to_core_ring
 add_pdn_connect -grid {grid} -layers {Metal1 Metal5}
 add_pdn_connect -grid {grid} -layers {Metal5 TopMetal1}
+# I/O pads
+add_pdn_connect -grid {grid} -layers {Metal5 TopMetal2}
+add_pdn_connect -grid {grid} -layers {TopMetal1 TopMetal2}
```

Comment:
@gadfort does this look right?  There seems to be some redundancy.

Reply:
It's a bit unusual since the I/O cells connect on TM2, the top-most layer, in both directions, disregarding the layer's routing direction and making it impossible to via upwards first.

Here are two screenshots comparing the results with and without the M5 <-> TM2 connection:

With:
![with](https://github.com/user-attachments/assets/230b2a16-f176-4bee-ab85-f3170ed8a0a8)

Without:
![without](https://github.com/user-attachments/assets/e93506f5-20eb-4aef-9217-460c4a60dd4c)

The difference is a little weird. Without the connection, there are far fewer but not zero horizontal connections.